rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -199,8 +199,7 @@ components:
             location/metadata/{metadataType} to store metadata, where {metadataType} is restricted
             to alphanumeric characters.
           type: string
-          minLength: 1
-          maxLength: 2048
+          format: uri
```

Comment:
I don't think this works like we would hope since no validator is provided out of the box. Unless that change includes a uri validator, we should hold off.

Reply:
I'm pretty sure it works by default. I created an itest in the ion-transformation repo and was able to reproduce the issue with `minLength`, `maxLength`, and `format: uri`. I included the exception in the PR description. The exception does not occur after removing `minLength` and `maxLength`.

Reply:
@cjlange, I'm replying to https://github.com/connexta/ion-transformation-api/pull/42#issuecomment-533130055 here to start a thread.

1. schema:
   ```yaml
    type: string
    minLength: 1
    maxLength: 2048
    ```
    `TransformRequest`:
    ```java
    @NotNull
    @Size(min=1,max=2048)
    public String getLocation() {
      return location;
    }

    @NotNull
    @Size(min=1,max=2048)
    public String getMetacardLocation() {
      return metacardLocation;
    }
    ```
2. schema:
    ```yaml
    type: string
    format: uri
    minLength: 1
    maxLength: 2048
    ```
    `TransformRequest`:
    ```java
    @NotNull
    @Valid
    @Size(min=1,max=2048)
    public URI getLocation() {
      return location;
    }

    @NotNull
    @Valid
    @Size(min=1,max=2048)
    public URI getMetacardLocation() {
      return metacardLocation;
    }
    ```
3. schema:
    ```yaml
    type: string
    format: uri
    ```
    `TransformRequest`:
    ```java
    @NotNull
    @Valid
    public URI getLocation() {
      return location;
    }

    @NotNull
    @Valid
    public URI getMetacardLocation() {
      return metacardLocation;
    }
    ```

Option 2 causes the error included in the PR description because [`@javax.validation.constraints.Size`](https://docs.oracle.com/javaee/7/api/javax/validation/constraints/Size.html) is only valid for
* `CharSequence` (length of character sequence is evaluated)
* `Collection` (collection size is evaluated)
* `Map` (map size is evaluated)
* Array (array length is evaluated)

Reply:
Regarding your second question, this is a breaking change, so we need to bump the version.

Reply:
What validation is occurring if minLength and maxLength aren't being defined and checked?